visas/letter.md

```diff
@@ -0,0 +1,34 @@
+![letterhead](./letterhead.svg)
+
+Toronto, ##date##
+
+To the Canadian Embassy in ##embassy.country##
+
+##embassy.address##
+##embassy.phone##
+
+**Subject: Invitation Letter for ##applicant.prefix## ##applicant.name##**
+
+Dear Sir/Madam,
+
+My name is ##organizer.name##, I was born on ##organizer.dob##, and I am from Toronto, Canada. I am a Canadian citizen and I live in ##organizer.address##, ##organizer.country##. I am a co-organizer of an annual conference in Toronto called Our Networks, which explores our technology practices and collective maintenance of our network infrastructures.
+
+This letter confirms that ##applicant.prefix## ##applicant.name##, who is ##applicant.nationality##, born on ##applicant.dob## and currently living in ##applicant.address##, ##applicant.country##, is invited to participate at Our Networks 2019, held from September 20–22, 2019 at the Toronto Media Arts Centre. Details about the event can be found at our conference website: https://ournetworks.ca
+
+##applicant.prefix## ##applicant.name## is currently employed as ##affiliation.title## by ##affiliation.name## in ##affiliation.country##. Our Networks is inviting ##applicant.prefix## ##applicant.name## to our conference to share ##applicant.pronoun## expertise and participate without monetary compensations from Our Networks. The cost of this trip, including flights, accommodation, and subsistence costs for the duration of ##applicant.pronoun## stay in Toronto, is covered by ##sponsor.name##, ##sponsor.description##.
+
+If the travel visa is approved, ##applicant.prefix## ##applicant.name## plans to arrive to Toronto a few days before the conference days and stay a couple days afterwards, depending on flight ticket arrangements. The total duration will fall within a two week period within September of 2019, given the conference is held from September 20-22. ##applicant.prefix## ##applicant.name## is a genuine visitor who is visiting Canada to participate at Our Networks 2019 and will not overstay ##applicant.pronoun## visa. Accommodation for the duration of ##applicant.pronoun## stay is arranged at my primary residence: ##organizer.address##
+
+Feel free to contact me should you have any further questions regarding myself or ##applicant.prefix## ##applicant.name##. You can also contact ##applicant.prefix## ##applicant.name## directly at ##applicant.phone## and ##applicant.email##
+
+Kind regards,
+
+![signature](./signature.png)
+##organizer.name##
+Our Networks
```

Comment:
I think you can cut this? Also not the official organizer address below, and you are sending letter as individual